src/d_deh.c

```diff
@@ -1964,7 +2008,14 @@ void deh_procThing(DEHFILE *fpin, FILE* fpout, char *line)
                   }
                   mi->splash_group = mi->splash_group + SG_END;
                 }
-              else if (ix == 31)
+              else if (ix == DEH_MOBJINFO_BLOODCOLOR)
+                {
+                  mobjinfo_t *mi = &mobjinfo[indexnum];
+                  mi->bloodcolor = (int)(value);
+                  if (mi->bloodcolor)
```

Comment:
I think we can move this check here:
https://github.com/fabiangreffrath/woof/blob/f9ebfefd3b9c2df9f35546b982537c005ecb51f5/src/v_video.c#L209-L210

Reply:
But in `V_BloodColor` we can't tell if the color change is from dehacked or from `D_SetBloodColor`.
But maybe there is a more elegant way to detect this.

Reply:
I just meant to move the range check here, where the value is applied.